test/cli/testutils.py

```diff
@@ -70,9 +78,31 @@ def __lookup_cppcheck_exe():
     return exe_path
 
 
+def __copy_and_prepare_cppcheck(tmpdir):
+    file_name = "cppcheck"
+    if sys.platform == "win32":
+        file_name += ".exe"
+    exe = __create_temporary_copy(tmpdir, file_name)
+
+    #add minimum cfg
+    if not os.path.exists(tmpdir + '/cfg'):
+        test_cfg_folder = tmpdir.mkdir('cfg')
+        test_cfg = test_cfg_folder.join('std.cfg')
+        if not os.path.exists(test_cfg):
```

Comment:
if condition at 88 is true it seems to me that this condition will always be true also.